corehq/apps/hqadmin/migrations/0002_vcmmigrationaudit.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
nit: in the future you can combine these all into 1 before PRing by rolling back the migrations, deleting them, and then rerunning makemigrations